plugins/woocommerce-admin/client/marketplace/components/product-card/product-card.scss

```diff
@@ -5,6 +5,50 @@
 		padding: $large-gap;
 		border-radius: $grid-unit-05 !important;
 
+		/* When product card is loading, contents will be empty and we render skeleton loader wireframes: */
+		&.is-loading {
+			overflow: hidden;
+			pointer-events: none;
+
+			.woocommerce-marketplace__product-card__content {
+				overflow: hidden;
+			}
+
+			.woocommerce-marketplace__product-card__title,
+			.woocommerce-marketplace__product-card__description,
+			.woocommerce-marketplace__product-card__price {
+				width: 100vw;
+			}
+
+			.woocommerce-marketplace__product-card__vendor,
+			.woocommerce-marketplace__product-card__description,
+			.woocommerce-marketplace__product-card__price {
+				animation: loading-fade 1.6s ease-in-out infinite;
+				background: repeating-linear-gradient(#f0f0f0, #f0f0f0 13px, transparent 13px, transparent 21px);
+				background-position-y: 4px;
+				height: calc(13px * 1.5);
+			}
+
+			.woocommerce-marketplace__product-card__icon {
+				background: #f0f0f0;
```

Comment:
`#f0f0f0` could be a variable.

Reply:
It absolutely should!

I had a mental note that I should do this. Then I didn't. It was getting late in the day and I was running around trying to make dog/child things happen! (Also why I didn't run the linter, oops!)